README.md

```diff
@@ -1,2 +1,55 @@
-# location-hours
-Poll the locations API for each library's regular hours as well as any closure alerts
+# LocationHoursPoller
+
+The LocationHoursPoller has two modes:
+* The `LOCATION_HOURS` mode periodically checks [Refinery](https://refinery.nypl.org/api/nypl/locations/v1.0/locations) for each library's regular hours, checks them against what's already stored in Redshift, and writes any newly updated hours to LocationHours Kinesis streams for ingest into the [BIC](https://github.com/NYPL/BIC)
+* The `LOCATION_CLOSURE_ALERTS` mode periodically checks [Refinery](https://refinery.nypl.org/api/nypl/locations/v1.0/locations) for any library closure alerts and writes them to LocationClosureAlert Kinesis streams for ingest into the [BIC](https://github.com/NYPL/BIC).
+
+## Running locally
+* `cd` into this directory
+* Add your AWS credentials (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) to the config file for the environment you want to run
```

Comment:
Would it be possible to authorize using `AWS_PROFILE` instead? So there's no risk of aws creds getting committed to a config file? It's also one fewer thing to configure.

Reply:
I can't actually test that this is possible yet, but I'm happy to assume this in the README for now